vm.go

```diff
@@ -544,7 +543,6 @@ func (vm *vm) init() {
 func (vm *vm) run() {
 	vm.halt = false
 	interrupted := false
-	ticks := 0
```

Comment:
Although it's trickier with removals, can we add a comment here like:

```
// DIVERSION: removed internal goja tick counter
```

This will be helpful in the future when we pull from upstream to have diversions readily called out.

Reply:
We could also comment out the code instead? and include the DIVERSION comment above?

Reply:
Should we still do this if ticks does not exist in master either?

Reply:
Ah nice I guess we can just remove it then, they use a profiler in future commits so this eventually goes away.